assets/js/script.js

```diff
@@ -0,0 +1,151 @@
+// Assignment Code
+var generateBtn = document.querySelector("#generate");
+// array for uppercase,lowercase,special characters and numbers.
+const numArray = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"];
+const specialArray = [
+  "!",
+  "£",
+  "$",
+  "%",
+  "^",
+  "&",
+  "*",
+  "(",
+  ")",
+  "_",
+  "-",
+  "+",
+  "=",
+  "?",
+  "@",
+  "#",
+  "~",
+  "|",
+  "/",
+  ".",
+  ",",
+  "<",
+  ">",
+  "'",
+  ":",
+  " ",
+  ";",
+  "`",
+  "¬",
+];
+const lowerCaseArray = [
+  "a",
+  "b",
+  "c",
+  "d",
+  "e",
+  "f",
+  "g",
+  "h",
+  "i",
+  "j",
+  "k",
+  "l",
+  "m",
+  "n",
+  "o",
+  "p",
+  "q",
+  "r",
+  "s",
+  "t",
+  "u",
+  "v",
+  "w",
+  "x",
+  "y",
+  "z",
+];
+
+// used a formula for uppercase array
+const upperCase = lowerCaseArray.join("").toUpperCase();
+const upperCaseArray = upperCase.split("");
+
```

Comment:
Interesting bit of code here, I'd not thought to do it this way. This does work so not a bad thing but I'd be mindful of, if something was wrong in the lower case array, it would also be wrong in the upper case array. I personally would of added in the capital letter array as it's own array rather than using this method, but this does work!